nginx-auto-config.go

```diff
@@ -225,3 +229,15 @@ func takeInput() int {
 	}
 	return input
 }
+
+func verifyDirInput() string {
+	dirName := getInput(false, false)
+
+	if !dirExists(dirName) {
+		_, _ = red.Printf("Directory '%v' is non existent, please try again.\n", dirName)
+		_, _ = cyan.Print("Root path: ")
+		return verifyDirInput()
+	} else {
+		return dirName
+	}
+}
```

Comment:
I'd suggest moving this to utils.go

Reply:
I initially thought of the same but the `takeInput` was defined in the main file. Hence this function too was defined in the same file. However, moved.